### ISSUE TITLE
what is your offline score

### ISSUE THREAD

**Issue by sanshanxiashi** [aerdem4/kaggle-quora-dup#5]
Hello！
First of all, thank you very much for your open source.
I am very interested in your implementation, so I downloaded your code and implemented it on my machine. My GPU is TiTanic Xp.
 After I finished running the file "model.py" with setting epochs=15, and I got the result of the validation set loss of about 0.203 (training set loss is about 0.17). The results of the model training don't seem so good!
I find you have a ranking of 23 on kaggle and an online score of 0.12988, so I would like to ask, what is your offline score? How can I use this open source code to achieve the same validation loss as you?

Looking forward to your reply.Thanks again!
@aerdem4

**Comment by aerdem4** [comment]
Hi, thanks for your question. The validation error you see is on one fold. When you run it for all the folds and take average of the models generated in each fold you get some gain from ensembling. In addition, if you look at postprocessing, I do some extra things to improve my score. As a last remark, logloss is a class balance sensitive metric, therefore logloss on train/val set and test set (Kaggle LB) can be very different since Kaggle didn't split them randomly.

**Comment by sanshanxiashi** [comment]
Thank you very much for your reply!
I understand what you mean. By the way, how does the correct rate of the test set be estimated? Is the probability greater than 0.9?

**Comment by sanshanxiashi** [comment]
Thank you very much for your reply!
I understand what you mean. By the way, how does the correct rate of the test set be estimated? Is the probability greater than 0.9?
@aerdem4

**Comment by aerdem4** [comment]
I think they probed it by submitting all 1s or all 0s.
